common/dtd/ldml.dtd

```diff
@@ -3164,6 +3164,36 @@ CLDR data files are interpreted according to the LDML specification (http://unic
 <!ATTLIST annotation draft (approved | contributed | provisional | unconfirmed) #IMPLIED >
     <!--@METADATA-->
 
+<!ELEMENT personNames ( alias | ( nameOrder*, personName+, special* ) ) >
+
+<!ELEMENT nameOrder EMPTY >
+<!ATTLIST nameOrder nameLocales NMTOKENS #REQUIRED >
+    <!--@MATCH:set/validity/locale-->
+<!ATTLIST nameOrder order (surnameFirst | givenFirst) #REQUIRED >
+<!ATTLIST nameOrder draft (approved | contributed | provisional | unconfirmed) #IMPLIED >
+    <!--@METADATA-->
+<!ATTLIST nameOrder references CDATA #IMPLIED >
+    <!--@METADATA-->
+
+<!ELEMENT personName ( alias | ( namePattern+, special* ) ) >
+<!ATTLIST personName length NMTOKENS #IMPLIED >
+    <!--@MATCH:set/literal/long, medium, short, monogram, monogram-narrow-->
```

Comment:
Do we need to make any changes for "long" to account for it being reserved word in many languages? @macchiati